### PR TITLE
ED-1237 - Add oci support

### PR DESCRIPTION
**Pull request by ddevadat**
For job 
Provision/KnowledgePlatform/Learning

### CODE REVIEW

File: platform-tools/spikes/content-tool/pom.xml [hunk 1]
```diff
@@ -66,10 +66,9 @@
         <dependency>
             <groupId>org.sunbird</groupId>
             <artifactId>cloud-store-sdk</artifactId>
-            <version>1.2.5</version>
+            <version>1.4.4</version>
```
Comment (maheshkumargangula) [inline platform-tools/spikes/content-tool/pom.xml:69]: @ddevadat - OCI supported with 1.4.6 right?

File: platform-modules/pom.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
 		<version.compiler.plugin>2.3.1</version.compiler.plugin>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<maven.compiler.source>1.8</maven.compiler.source>
-		<cloud.store.version>1.2.8</cloud.store.version>
+		<cloud.store.version>1.4.4</cloud.store.version>
```
Comment (maheshkumargangula) [inline platform-modules/pom.xml]: @ddevadat - OCI supported with 1.4.6 right?

Comment (ddevadat) [reply platform-modules/pom.xml]: corrected the version. Do i need to update the version in spike module also? i am not sure if its in use

**Comment by maheshkumargangula** [comment]
@ddevadat - Please handle the conflicts.
Cc: @pallakartheekreddy

**Comment by ddevadat** [comment]
no longer required. closing this pr

**Pull request closed without merging**